src/modules/user/food/components/AddonsCard.tsx

```diff
@@ -19,7 +19,7 @@ const AddonsCard: React.FC<AddonsCardProps> = ({ addon }) => {
         <Checkbox onChange={(e) => handleChange(e, addon.title)}>
           {addon.title}
         </Checkbox>
-        <Text type="secondary">+{addon.price} Baht</Text>
+        <Text type="secondary">+฿{addon.price}</Text>
```

Comment:
This should be +100฿ ?
![image](https://github.com/meeorder/meeorder-frontend/assets/64889751/61a75301-d0f8-4e11-8587-534e6d2edd66)

Reply:
From my 10 minutes of research and sleeping. I've found out that any currency symbol including `฿` must be placed `before` the amount.

References
- [This goofy blog](http://p-warid.blogspot.com/2015/08/bath-vs-baht.html#:~:text=*%20%E0%B9%80%E0%B8%A3%E0%B8%B2%E0%B8%AA%E0%B8%B2%E0%B8%A1%E0%B8%B2%E0%B8%A3%E0%B8%96%E0%B9%83%E0%B8%8A%E0%B9%89%20THB%20%E0%B8%AB%E0%B8%A3%E0%B8%B7%E0%B8%AD,%E0%B9%80%E0%B8%8A%E0%B9%88%E0%B8%99%20THB%20350%2C%20%E0%B8%BF%2020)
- [Oh no it's Pantip](https://pantip.com/topic/36002237)
- [There aren't many Thai forums on this discussion so imma ask my best bro](https://chat.openai.com/share/3453a4cd-c042-4c9f-bffd-cd90b31a4874)